x/distribution/spec/03_begin_block.md

```diff
@@ -0,0 +1,80 @@
+<!--
+order: 3
+-->
+
+# Begin Block
+
+At each `BeginBlock`, the fees received in the previous block are transferred to the distribution `ModuleAccount`, as it's the account the one who keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve community tax is then charged, then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Note the social distribution is applied to proposer validator in addition to the proposer reward.
```

Comment:
```suggestion
At each `BeginBlock`, portion of the fees received in the previous block are transferred to the distribution `ModuleAccount` - the account which keeps track of the coin flow for the distribution. The reminder of the fees are allocated to the proposer, community fund and global pool. The block proposer of the previous round (and it's delegators) also receives between 1% and 5% of fee rewards, the reserve community tax is then charged, then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Note the social distribution is applied to proposer validator in addition to the proposer reward.
```

Reply:
One correction: all the fees are sent to the ModuleAccount, and stay there until they are withdrawn. All the remaining operations are about updating the module-internal bookkeeping. That bookkeeping is used to send out the right amounts at withdraw time. Like a bank which has all its money in a big vault, but has books that keep track of which customer has which balance.

Other than that, I like the bullet style, I'll add that.